flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -283,6 +283,25 @@ public void testTransformationSetParallelism() {
         assertThat(vertices.get(2).isParallelismConfigured()).isTrue();
     }
 
+    @Test
+    public void testTransformationSetMaxParallelism() {
```

Comment:
All test methods in junit5 should be package-private.
```java
void testTransformationSetMaxParallelism() 
```

Reply:
Junit 4 requires all test methods to be public. We are currently migrating to Junit5, which does not have this requirement. And then to keep visibility to a minimum, it's better to make test methods package-private. 
There are 5 test methods that are public, others are package-private. I guess it may be that the visibility was not noticed when these test methods were introduced. 

Anyway, it's better to keep the same style in test cases. Would you mind changing them to package-private in a separate hotfix commit in this PR?

Reply:
Sounds good, added a hotfix commit.